smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/StructureGenerator.kt

```diff
@@ -221,30 +223,35 @@ class StructureGenerator(
     object AdditionalErrorMembers : SectionId
 
     private fun generateErrorStructMembers() {
```

Comment:
Modeled error properties are now exposed using a structure exposed as a instance var named `properties`.
Most of the codegen changes you will see are related to namespacing error properties (and not input/output structs) under the `properties` instance var.